pkgs/by-name/sn/sn-pro/package.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "sn-pro";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "supernotes";
+    repo = pname;
+    rev = "${version}";
```

```suggestion
    rev = version;
```

unnecessary string interpolation.